example_chat_completion.py

```diff
@@ -93,7 +93,7 @@ def _fn(
     max_seq_len: int = 512,
     max_batch_size: int = 4,
     max_gen_len: Optional[int] = None,
-    dynamo: bool = True,
+    dynamo: str = "inductor",
```

Comment:
why do we set the default ot inductor? Shouldn't default be `openxla_eval` to preserve the existing behavior?

Reply:
Either way would preserve the existing behavior because the behavior depends on `USE_CUDA` as well as on `dynamo`. :(

The existing behavior was:
- `dynamo = True` (default). Then, depending on `USE_CUDA`:
  - if `USE_CUDA`, we'd pick Inductor by not passing `backend=` to `torch.compile` (`backend=inductor` is the default)
  - else, we'd pick `openxla_eval` by passing `backend=openxla_eval` to `torch.compile`.
- else: eager mode.

Now we still need to fiddle with `USE_CUDA`, but we can specify `openxla` whereas before we could not. Note that now we have to pass `-inductor ''` (instead of `-inductor False`) to use eager mode.

Reply:
+1 
not sure if this UX is most effective. Wdyt we find a way to maintain the default behavior as openxla?

Reply:
We can just set the default to "openxla_eval". But note that we still need to fiddle with `USE_CUDA` etc. I won't have time until after the break, but I think it's possible to make everything work without relying on `USE_CUDA` (other than for it to signal "use CUDA", as opposed to "use Inductor"). Will look into it.

Reply:
PTAL -- see my comment below https://github.com/pytorch-tpu/llama/pull/47#issuecomment-1920150636